.github/workflows/build_and_test.yml

```diff
@@ -282,6 +282,9 @@ jobs:
   lint:
     name: Linters, licenses, dependencies and documentation generation
     runs-on: ubuntu-20.04
+    env:
+      LC_ALL: C.UTF-8
+      LANG: C.UTF-8
```

Comment:
Ur, interesting. Previously, I moved this from here to line 360 because the linter is executed inside `dongjoon/apache-spark-github-action-image:20201025`.